compiler/src/iree/compiler/Codegen/Interfaces/BufferizationInterfaces.cpp

```diff
@@ -46,141 +46,12 @@ using mlir::bufferization::replaceOpWithBufferizedValues;
 using mlir::bufferization::replaceOpWithNewBufferizedOp;
 
 namespace mlir::iree_compiler {
-
-//===----------------------------------------------------------------------===//
-// Utility functions.
-//===----------------------------------------------------------------------===//
-
-/// Get strides for row-major oredering of a tensor with the given `shape`.
-static SmallVector<int64_t> getStridesFromShape(ArrayRef<int64_t> shape) {
-  if (shape.empty()) {
-    return {};
-  }
-  SmallVector<int64_t> strides(shape.size(), ShapedType::kDynamic);
-  strides.back() = 1;
-  for (int i = strides.size() - 1; i > 0; --i) {
-    if (ShapedType::isDynamic(shape[i])) {
-      break;
-    }
-    strides[i - 1] = strides[i] * shape[i];
-  }
-  return strides;
-}
-
-static MemRefType
-getMemrefTypeForTensor(IREE::TensorExt::DispatchTensorType tensorType,
-                       MemRefLayoutAttrInterface layout = {},
-                       Attribute memorySpace = {}) {
-  return MemRefType::get(tensorType.getShape(),
-                         tensorType.getBoundElementType(), layout, memorySpace);
-}
-
-/// Find the memref version of the given InterfaceBindingSubspanOp. If no such
-/// op exists in the same block (before the given op), create a new op.
-// TODO(#12933): Because of regressions in CUDA backend, there is an
-// option to keep a legacy mode of not representing the offset in the
-// type. Remove once the bug is fixed.
```

Comment:
I verified that the TODO was cleared in https://github.com/iree-org/iree/commit/784b1220580a28254324cc1878e8ae6cb3783017, so removing it in this refactor looks okay to me.

Reply:
Let's open this conversion for visibility. Just in case if others have the same confusion when they review the change.